starcheck/plot.py

```diff
@@ -0,0 +1,187 @@
+import os
```

Comment:
This module is largely copied in from mica, but will probably end up living in starcheck when complete.